config/config.exs

```diff
@@ -13,6 +13,7 @@ config :skate,
   # Default. Can be configured via environment variable, which is loaded in application.ex
   api_url: {:system, "API_URL"},
   api_key: {:system, "API_KEY"},
+  restrict_environment_access?: System.get_env("RESTRICT_ENVIRONMENT_ACCESS?") == "true",
```

Comment:
Based on the following links, I think the `?` should be removed
- https://stackoverflow.com/a/2821201
- https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_10_02
- https://stackoverflow.com/a/2821183


> > Environment variable names used by the utilities in the Shell and Utilities volume of IEEE Std 1003.1-2001 consist solely of uppercase letters, digits, and the '_' (underscore) from the characters defined in Portable Character Set and do not begin with a digit. Other characters may be permitted by an implementation; applications shall tolerate the presence of such names.
> 
> So while the names may be valid, your shell might not support anything besides letters, numbers, and underscores.